File: src/main/java/me/lfojacintho/hellofresh/deliverycheck/HellofreshDeliveryCheckApplication.java
package me.lfojacintho.hellofresh.deliverycheck;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan("me.lfojacintho.hellofresh.deliverycheck")
public class HellofreshDeliveryCheckApplication {

	public static void main(String[] args) {
			SpringApplication.run(HellofreshDeliveryCheckApplication.class, args);
	}
}

File: src/main/kotlin/me/lfojacintho/hellofresh/deliverycheck/HelloFreshDeliveryCheckApplication.kt
package me.lfojacintho.hellofresh.deliverycheck

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan("me.lfojacintho.hellofresh.deliverycheck")
open class HellofreshDeliveryCheckApplication

fun main(args: Array<String>) {
    runApplication<HellofreshDeliveryCheckApplication>(*args)
}

